frontend/src/App.jsx

```diff
@@ -46,15 +46,12 @@ function App() {
     <div className="App">
       {isShown && <FilterPage eventFilterButton={eventFilterButton} />}
       <Filter />
+      <Leaflet fuelList={fuelList} geo={pointGeo} />
       <button type="button" onClick={() => changeView()}>
         {visible ? "⇩" : "⇧"}
       </button>
-      <StationListing fuelList={fuelList} visible={visible} />
-      <Leaflet fuelList={fuelList} geo={pointGeo} />
+      {visible ? <StationListing fuelList={fuelList} /> : " "}
```

Comment:
If you have nothing else to display, replace your ternary operator byr && operator.